src/components/Nav.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { PrimaryNav, MenuLink, Menu } from "./NavElements";
+import { constants } from "../routes/constants";
+import chatIcon from "../assets/imgs/chat-icon.png";
+function Nav() {
+  const [userPicture, setUserPicture] = useState(
+    "https://www.w3schools.com/howto/img_avatar.png"
```

Comment:
u can download this image and save it in assets file

Reply:
This is not the real pic. the pic will come from the backend so I prefer to save it on state right now